src/integrationTest/resources/logback-test.xml

```diff
@@ -8,6 +8,8 @@
     </appender>
     <logger name="org.hibernate" level="warn" additivity="true"/>
     <logger name="org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator" level="debug" additivity="true"/>
+    <logger name="org.hibernate.SQL" level="debug" additivity="true"/>
+    <logger name="org.hibernate.orm.jdbc.bind" level="trace" additivity="true"/>
     <logger name="org.mongodb.driver" level="warn" additivity="true"/>
     <logger name="com.mongodb.hibernate" level="debug" additivity="true"/>
     <root level="info">
```

Comment:
as per https://docs.jboss.org/hibernate/orm/6.6/userguide/html_single/Hibernate_User_Guide.html#best-practices-logging. In next ResultSet PR another `org.hibernate.orm.jdbc.extract` will be added.